test/MongoUtil_test.py

```diff
@@ -62,9 +62,11 @@ def test_init_ok(self):
         self.assertEqual(handle_collection.name, 'handle')
         self.assertEqual(handle_collection.count_documents({}), 10)
 
-        handle_col_idx = handle_collection.index_information().keys()
+        handle_col_info = handle_collection.index_information()
+        handle_col_idx = handle_col_info.keys()
         expected_idx = ['_id_', 'hid_1']
         self.assertCountEqual(handle_col_idx, expected_idx)
```

Comment:
Sorry, I missed this before - why assertCountEqual vs assertEqual?

Reply:
the `_id` is automatically indexed. Do you want me to change it to something like assertIn('hid_1', handle_col_idx) to only check `hid` is indexed?

Reply:
oh I think I see what you mean now. `handle_col_idx` was a dictkey. I was just lazy to cast it to a list. I can change it to use assertEqual.

Reply:
👍